src/main/kotlin/no/skatteetaten/aurora/gobo/integration/mokey/applicationResources.kt

```diff
@@ -35,6 +36,8 @@ data class ApplicationDeploymentDetailsResource(
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class ApplicationDeploymentResource(
+    val identifier: String,
```

Comment:
Out of curiosity, why not use short form?

Reply:
I'm not using the short form because the ResourceSupport classes from spring-hateoas reserves the id property name for the self link, and is actually explicitly excluded from Json serialization/deserialization. So instead of fighting the framework on both the way in and out, I decided to just call the field something else. Once out of the *Resource classes, I revert back to using the more conventional id name for the property.